src/Altinn.App.Core/Infrastructure/Clients/Storage/ProcessClient.cs

```diff
@@ -46,7 +46,7 @@ public class ProcessClient : IProcessClient
         /// <inheritdoc/>
         public Stream GetProcessDefinition()
         {
-            string bpmnFilePath = _appSettings.AppBasePath + _appSettings.ConfigurationFolder + _appSettings.ProcessFolder + _appSettings.ProcessFileName;
+            string bpmnFilePath = Path.Combine(_appSettings.AppBasePath , _appSettings.ConfigurationFolder , _appSettings.ProcessFolder , _appSettings.ProcessFileName);
```

Comment:
```suggestion
            string bpmnFilePath = Path.Join(_appSettings.AppBasePath , _appSettings.ConfigurationFolder , _appSettings.ProcessFolder , _appSettings.ProcessFileName);
```
https://learn.microsoft.com/en-us/dotnet/api/system.io.path.combine?view=net-8.0#remarks

> This method is intended to concatenate individual strings into a single string that represents a file path. However, if an argument other than the first contains a rooted path, any previous path components are ignored, and the returned string begins with that rooted path component. As an alternative to the Combine method, consider using the [Join](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.join?view=net-8.0) or [TryJoin](https://learn.microsoft.com/en-us/dotnet/api/system.io.path.tryjoin?view=net-8.0) methods.